src/examples/learn_quant/measures.py

```diff
@@ -0,0 +1,208 @@
+from ultk.effcomm.informativity import informativity
+from ultk.language.grammar import GrammaticalExpression
+from ultk.language.language import Language, aggregate_expression_complexity
+from ultk.language.semantics import Meaning
+
+from learn_quant.meaning import create_universe
+from learn_quant.util import read_expressions
+from learn_quant.quantifier import QuantifierUniverse, QuantifierModel
+from concepts.contexts import Context
+
+from itertools import product
+import math
+from scipy.stats import entropy
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+from scipy import sparse
+
+
+class MonotonicityMeasurer:
```

Comment:
Can we walk through this in our next in-person meeting?  It oculd use a bit more documentation, and there are a few things I'm unsure of (e.g. the use of a DF and the logic in `_get_truth_matrix`)